src/plugins/simpleini/simpleini.c

```diff
@@ -18,33 +18,48 @@
 #include <kdbease.h>
 #include <kdberrors.h>
 #include <kdblogger.h>
+#include <kdbprivate.h>    // elektraStrNDup
```

Comment:
Could you please move the forward declaration to `kdbhelpers.h` and avoid including kdbprivate?